misk-aws/src/main/kotlin/misk/dynamodb/DynamoDbHelpers.kt

```diff
@@ -0,0 +1,40 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import java.time.Duration
+import misk.backoff.Backoff
+import misk.backoff.DontRetryException
+import misk.backoff.ExponentialBackoff
+import misk.backoff.retry
+
+object DynamoDbHelpers {
+  val DEFAULT_BACKOFF = ExponentialBackoff(
+      baseDelay = Duration.ofMillis(10),
+      maxDelay = Duration.ofMillis(500),
+      jitter = Duration.ofMillis(5)
```

Comment:
We may need larger ``jitter`` to decrease the chance of time conflicts, how about a ``jitter`` of 10?